### ISSUE TITLE
Firmware update

### ISSUE THREAD

**Issue by ArvindSami** [modmypi/PiModules#44]
Trying to update the firmware 

enter the following:

sudo python 9600_picofuHV3.0.py 0xE3_UPS_PIco_HV3.0_main.hex

return is 

python: can't open file 9600_picofuHV3.0.py [Er no2] No such file or directory

**Comment by Nio9345** [comment]
Yeah, I have the same problem and it seems noone gives a damn about it. Wasted around 30 bucks and I can't update the firmware and the fan isn't working (the motor just vibrates and doesn't turn).

**Comment by Cesar-DC** [comment]
Hello,

I think this is the file you are looking for:
https://github.com/uziel123/modules

Regards,
